### DISCUSSION TITLE
Osmand offline license?

### DISCUSSION THREAD

**Discussion by Suncatcher** [osmandapp/OsmAnd#14751] [Q&A]
Is there a way I can use Osmand on the another account than it was purchased? Can I convert Play Store license into offline one and use Osmand on the phone without Google services?

**Comment by vshcherb** [comment]
You can use nightly builds or F-Droid build

**Reply by Suncatcher** [reply]
how the nightly build help me to turn it into Osmand+? I am asking about paid version

**Reply by vshcherb** [reply]
Then we don't have any offline license, all purchases are distributed through the markets i.e. Google Play / Amazon / Huawei.
